bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/GeyserStandaloneBootstrap.java

```diff
@@ -127,6 +128,21 @@ public void onEnable() {
         }
     }
 
+    /**
+     * Check using {@link java.awt.GraphicsEnvironment} that we are a headless client
+     *
+     * @return If the current environment is headless
+     */
+    private boolean checkHeadless() {
+        try {
+            Class<?> graphicsEnv = Class.forName("java.awt.GraphicsEnvironment");
+            Method isHeadless = graphicsEnv.getDeclaredMethod("isHeadless");
+            return (boolean)isHeadless.invoke(null);
+        } catch (Exception ignore) { }
+
+        return true;
```

Comment:
Does this affect the GUI from running? This would return false when ran with the code in line 64 if the above try/catch code fails.

Reply:
If `GraphicsEnvironment` is missing then we are headless so it should default true.